apps/etch/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# F E T C H   V M   L A N G
+#
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+project(fetch-etch)
```

Comment:
Can we just call the project `etch`? Constellation does not have the `fetch-` prefiix and it's caused no issues.